.gitattributes

```diff
@@ -0,0 +1,2 @@
+components/sup-protocol/src/generated/*.rs linguist-generated
```

Comment:
The example at https://github.com/github/linguist#generated-code has `linguist-generated=true`.  Is the `=true` required?

Reply:
Probably not; I took my inspiration from https://robots.thoughtbot.com/github-diff-supression, but I guess we'll see.